element_lab/project_with_study.py

```diff
@@ -0,0 +1,102 @@
+import importlib
+import inspect
+
+import datajoint as dj
+
+from element_lab import lab
+
+schema = dj.Schema()
+
+_linking_module = None
+
+
+def activate(
+    project_schema_name,
+    lab_schema_name,
+    *,
+    create_schema=True,
+    create_tables=True,
+    linking_module=None,
+):
+    """
+    activate(schema_name, create_schema=True, create_tables=True, linking_module=None)
+        :param schema_name: schema name on the database server to activate the
+                            `lab` element
+        :param create_schema: when True (default), create schema in the
+                              database if it does not yet exist.
+        :param create_tables: when True (default), create tables in the
+                              database if they do not yet exist.
+        :param linking_module: a module (or name) containing the required
+                            dependencies to activate the `event` element:
+    """
+    if isinstance(linking_module, str):
+        linking_module = importlib.import_module(linking_module)
+    assert inspect.ismodule(linking_module), (
+        "The argument 'dependency' must" + " be a module or module name"
+    )
+
+    global _linking_module
+    _linking_module = linking_module
+
+    lab.activate(
+        lab_schema_name,
+        create_schema=create_schema,
+        create_tables=create_tables,
+        linking_module=_linking_module,
+    )
+
+    schema.activate(
+        project_schema_name,
+        create_schema=create_schema,
+        create_tables=create_tables,
+        add_objects=_linking_module.__dict__,
+    )

```

Comment:
I believe we were keeping a 1-to-1 relationship between a schema and a module.  So perhaps we need to make this module a new schema?